meilisearch/src/routes/mod.rs

```diff
@@ -260,32 +259,19 @@ pub fn create_all_stats(
     let mut last_task: Option<OffsetDateTime> = None;
     let mut indexes = BTreeMap::new();
     let mut database_size = 0;
-    let processing_task = index_scheduler.get_tasks_from_authorized_indexes(
-        Query { statuses: Some(vec![Status::Processing]), limit: Some(1), ..Query::default() },
-        filters,
-    )?;
+
     // accumulate the size of each indexes
-    let processing_index = processing_task.first().and_then(|task| task.index_uid());
-    index_scheduler.try_for_each_index(|name, index| {
-        if !filters.is_index_authorized(name) {
-            return Ok(());
+    for index_uid in index_scheduler.index_names()? {
+        if !filters.is_index_authorized(&index_uid) {
+            continue;
         }
 
-        database_size += index.on_disk_size()?;
```

Comment:
Ho! you removed the increment of the database size.

But, don't worry, that's actually great, you'll be able to **always** increment it even if the user doesn't have the right to access this index 😊 We talked about that with @gmourier and thought it was bad to not show the real database size on disk, this is not a security concern and it conflicts with the [Principle of least Astonishment](https://en.wikipedia.org/wiki/Principle_of_least_astonishment). Could you change that in this PR, please?

Reply:
Isn't it the kind of thing that allows a [side-channel attack](https://en.wikipedia.org/wiki/Side-channel_attack)?

Personally, I don't have any strong opinion on this subject, but the fact that this line was removed is definitely a bug 😂

Reply:
This is not a security issue but more a correctness issue here. The `databaseSize` stats value should display the actual database size, not less. I don't think there is any possible attack with this `/stats` route.

Reply:
Thank you @dureuill 😂

Reply:
done